.vscode/settings.json

```diff
@@ -1,10 +1,4 @@
 {
-    "dart.lineLength": 140,
-    "[dart]": {
-        "editor.rulers": [
-            140
-        ],
-    },
     "cmake.configureOnOpen": false,
```

Comment:
This line can probably get nixed. I think I accidentally clicked on something that popped up in VSCode like "hey did you know there's a marketplace extension for this file" and i was like STOP

Reply:
👋